js/ng-views/edit.html

```diff
@@ -0,0 +1,8 @@
+<h2>Edit Entry</h2>
+
+<form>
+  <input "Author" value="vm.entry.author" data-ng-model="vm.entry.author" />
```

Comment:
You don't need both `value` and `ng-model`. `ng-model` alone will take care of populating the input field.